### DISCUSSION TITLE
Slash Commands in a cog

### DISCUSSION THREAD

**Discussion by CSutter5** [Rapptz/discord.py#7747] [Q&A] [answered]
How do you create a slash command that is also inside of a cog?

**Comment by gsmj** [comment, marked as answer]
> How do you create a slash command that is also inside of a cog?

https://gist.github.com/AbstractUmbra/a9c188797ae194e592efe05fa129c57f

https://discordpy.readthedocs.io/en/master/ (v2.0 docs)

**Reply by CSutter5** [reply]
Ok, thank you

**Reply by JDJGInc** [reply]
I would recommend joining the guild, if you haven't for any future help needs
